EIPS/eip-1.md

```diff
@@ -363,8 +490,8 @@ This is a sentence with a footnote.[^1]
 
 Which renders to:
 
-<!-- markdownlint-capture -->
-<!-- markdownlint-disable code-block-style -->
+<!--- markdownlint-capture --->
+<!--- markdownlint-disable code-block-style --->
```

Comment:
Why the extra dash?

Reply:
The dashes are from an ill-advised global search-and-replace.  I've fixed them, but the bot, in it's infinite wisdom, suddenly decided that the DOI example -- which has been broken since 2022-- must now be fixed before I can check in my changes.  And since I am a second-class editor I can't override it.

Reply:
Following is the error that just popped up.  I cannot find any difference to explain why it suddenly complained.

Reply:
```
This is a sentence with a footnote.[^1]

[^1]:
    ```csl-json
    {
 Check failure on line 500 in EIPS/eip-1.md
GitHub
Actions
/ Markdown Linter

Code block style [Expected: fenced; Actual: indented]

EIPS/eip-1.md:500 MD046/code-block-style Code block style [Expected: fenced; Actual: indented]
      "type": "article",
      "id": 1,
      "author": [
```

Reply:
It's always complained about this.

Reply:
Also, the markdownlint bot isn't a required check.

Reply:
OK, thanks.  It does seem to be generating bad output at the end of the document, and I don't know how I broke it.